intranet/templates/oauth2_provider/authorize.html

```diff
@@ -61,6 +61,9 @@ <h3 class="block-center-heading">A third-party application wants access to your
                 </div>
             </form>
 
+        {% elif request.GET.client_id in DJANGO_SETTINGS.IOS_APP_CLIENT_IDS %}
```

Comment:
How about if `client_id`does not exist in`request.GET`?

Reply:
Testing indicates that it fails gracefully. The Django template language is designed to fail silently in a lot of cases like this (which sometimes is helpful and sometimes is not :slightly_smiling_face:).

More specifically, it appears to evaluate to `None` (adding `None to `IOS_APP_CLIENT_IDS` and visiting the page without passing `client_id` causes the message to appear).